src/util/ALE.sol

```diff
@@ -326,60 +307,115 @@ contract ALE is Ownable, ReentrancyGuard, CurveDBRHelper {
         dola.approve(spender, _value);
     }
 
+    /// @notice Borrow DOLA on behalf of the user
+    /// @param _value Amount of DOLA to borrow
+    /// @param _permit Permit data
+    /// @param _dbrData DBR data
+    /// @param market The market contract
+    function _borrowDola(
+        uint256 _value,
+        Permit calldata _permit,
+        DBRHelper calldata _dbrData,
+        IMarket market
```

Comment:
Should we authenticate the market?

Reply:
Are you referring to check if a market is valid? If that's the case we already authenticate it here 
https://github.com/InverseFinance/FiRM/blob/7d756d96d4223c95e2e039a6cc255cc7423f432a/src/util/ALE.sol#L134